misc/docerina/src/main/java/org/ballerinalang/docgen/generator/model/ClassPageContext.java

```diff
@@ -16,13 +16,15 @@
 package org.ballerinalang.docgen.generator.model;
 
 /**
- * Page context for the object page.
+ * Page context for the bClass page.
+ *
+ * @since 2.0
  */
-public class ObjectPageContext extends ModulePageContext {
-    public Object object;
-    public ObjectPageContext(Object object, Module module, Project project, String rootPath, String title,
-                             boolean excludeIndex) {
+public class ClassPageContext extends ModulePageContext {
+    public BClass bClass;
```

Comment:
```suggestion
    private BClass bClass;
```
Access ca be private

Reply:
I think this has to be kept public because its accessed by handlebars template